lxc/remote.go

```diff
@@ -170,7 +170,7 @@ func (c *cmdRemoteAdd) RunToken(server string, token string, rawToken *api.Certi
 
 		conf.Remotes[server] = config.Remote{Addr: addr, Protocol: c.flagProtocol, AuthType: c.flagAuthType, Domain: c.flagDomain}
 
-		d, err = conf.GetInstanceServer(server)
+		_, err = conf.GetInstanceServer(server)
```

Comment:
If d isn't used outside of the error check can we move the definition of the var inside too.

Reply:
Looks like `d` is used outside of the error check but not outside of the for loop so I've removed the definition at the function top level and used a `:=` inside the loop.